pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -80,6 +83,28 @@ func getStorageInitializerConfigs(configMap *v1.ConfigMap) (*StorageInitializerC
 	return storageInitializerConfig, nil
 }
 
+func getContainerSpecForStorageUri(storageUri string, client client.Client) (*v1.Container, error) {
+	storageContainers := &v1alpha1.ClusterStorageContainerList{}
+	if err := client.List(context.TODO(), storageContainers); err != nil {
+		return nil, err
+	}
+
+	for _, sc := range storageContainers.Items {
+		if sc.IsDisabled() {
```

Comment:
skip checking if there is a match if disabled.
We can introduce a priority field in the future.